container-engine-lib/lib/backend_impls/docker/docker_manager/create_and_start_container_args.go

```diff
@@ -1,9 +1,10 @@
 package docker_manager
 
 import (
+	"net"
```

Comment:
I don't think this `net` is needed for file anymore no?

Reply:
Still needed for `net.IP` I believe